src/platform/mock-radio.ts

```diff
@@ -3,6 +3,11 @@
  */
 
 export interface MockUDPListener {
```

Comment:
do we need to keep this? given that it's not being used anymore?

Reply:
Removed `mock-radio.ts` for this PR since nothing is being used yet.  I anticipate adding it back for #23 and #16